src/EditorFeatures/Core/Implementation/CodeFixes/CodeFixService.cs

```diff
@@ -588,7 +615,7 @@ private bool IsInteractiveCodeFixProvider(CodeFixProvider provider)
 
         private static readonly Func<DiagnosticId, List<CodeFixProvider>> s_createList = _ => new List<CodeFixProvider>();
 
-        private ImmutableArray<DiagnosticId> GetFixableDiagnosticIds(CodeFixProvider fixer, IExtensionManager extensionManager)
+        private ImmutableArray<DiagnosticId> GetFixableDiagnosticIds(CodeFixProvider fixer, IExtensionManager? extensionManager)
```

Comment:
❔ Can this ever be null in practice? An implementation of `IExtensionManager` is provided in the workspaces layer, so there is no reason why it should ever _need_ to be missing.

Reply:
Yes, there is an explicit comment and null check right below this line.

Reply:
I know the method _allowed_ null, but is there a need to do that? Can we just eliminate the special handling for null and simplify everything based on the callers all being non-null?

Reply:
That was adding a non-trivial and unrelated churn to this code. I'd rather we do it as a separate cleanup.